sdmetrics/reports/multi_table/_properties/column_pair_trends.py

```diff
@@ -0,0 +1,17 @@
+"""Column pair trends property for multi-table."""
+from sdmetrics.reports.multi_table._properties import BaseMultiTableProperty
+from sdmetrics.reports.single_table._properties import (
+    ColumnPairTrends as SingleTableColumnPairTrends)
+
+
+class ColumnPairTrends(BaseMultiTableProperty):
+    """Column pair trends property for multi-table.
+
+    This property evaluates the matching in trends between pairs of real
+    and synthetic data columns. Each pair's correlation is calculated and
+    the final score represents the average of these measures across all column pairs
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._single_table_property = SingleTableColumnPairTrends
```

Comment:
I think that this can be removed and have just `_single_table_property = SingleTableColumnPairTrends` as  attribute of the class and this will still work, no need for the `__init__`

Reply:
Yes it would be easier this way, done in [61510e9](https://github.com/sdv-dev/SDMetrics/pull/374/commits/61510e99ff095d8a955a9ab6c1942cb42002d044)